_posts/2018-03-01-core-team-announcement.md

```diff
@@ -31,4 +30,4 @@ But every change brings new opportunity, and the Core Team is delighted to welco
 All members of the Core Team, past and present want to thank all volunteers and contributors for their consistent and tireless work. We look forward to what 2018 has in store for us as a project, and as a community.
 
 ![Monero Core Team Cartoon](/img/monero-core.jpg)
-Image credit: [helloluis](https://cryptopop.net/)
\ No newline at end of file
+Image credit: [helloluis](https://cryptopop.net/)

```

Comment:
Why does this show as changed?

Reply:
It added a newline to the end of the file.

Reply:
>Why does this show as changed?

I used vim on linux and the file was most likely created in windows. I can try to revert if needed.